helm/portieris/templates/NOTES.txt

```diff
@@ -1,5 +1,4 @@
 Portieris is installed in your cluster.
 
-Default security policies are installed in your cluster. You can modify the default policies or replace them with your own. For more information, see http://ibm.biz/cise_default.
+You should update the policies according your your local requirements, for more information see https://github.com/IBM/portieris/blob/master/POLICIES.md
```

Comment:
Repeated word: "according your your local requirements"

Possibly this was intended to say "according to your local requirements"?